sdk/communication/AzureCommunicationCommon/Source/Identifiers.swift

```diff
@@ -302,70 +292,55 @@ public func createCommunicationIdentifier(fromRawId rawId: String) -> Communicat
 }
 
 /**
- Communication identifier for Microsoft bots.
+ Communication identifier for Microsoft Teams applications.
  */
-@objcMembers public class MicrosoftBotIdentifier: NSObject, CommunicationIdentifier {
-    public let botId: String
-    public let isResourceAccountConfigured: Bool
+@objcMembers public class MicrosoftTeamsAppIdentifier: NSObject, CommunicationIdentifier {
+    public let teamsAppId: String
     public let cloudEnvironment: CommunicationCloudEnvironment
     public var rawId: String
-    public var kind: IdentifierKind { return .microsoftBot }
+    public var kind: IdentifierKind { return .microsoftTeamsApp }
 
     /**
-     Creates a MicrosoftBotIdentifier object
-     - Parameter botId: The unique Microsoft app ID for the bot as registered with the Bot Framework.
-     - Parameter isResourceAccountConfigured: Set this to true if the bot is tenantized.
-                                It is false if the bot is global and no resource account is configured.
-     - Parameter cloudEnvironment: The cloud that the Microsoft Bot belongs to.
+     Creates a MicrosoftTeamsAppIdentifier object
+     - Parameter teamsAppId: The id of the Microsoft Teams application.
+     - Parameter cloudEnvironment: The cloud that the Microsoft Teams application belongs to.
                                     A null value translates to the Public cloud.
      */
     public init(
-        botId: String,
-        isResourceAccountConfigured: Bool = true,
+        teamsAppId: String,
         cloudEnvironment: CommunicationCloudEnvironment = .Public
     ) {
-        self.botId = botId
-        self.isResourceAccountConfigured = isResourceAccountConfigured
+        self.teamsAppId = teamsAppId
```

Comment:
About naming, in the end we went with appId for all languages except for javascript, and they had to use the teamsAppId, because of some name conflicts specific to JS. If possible, we should use appId instead of teamsAppId to be consistent.

Reply:
Ok, renamed to appId.